topics/command-tips.md

```diff
@@ -0,0 +1,88 @@
+# Command tips
+
+An @array-reply of strings, command tips are meant to provide additional information about commands for clients and proxies.
+This infromation should help clients and proxies to determine how to handle this command when Redis is in clustering mode.
```

Comment:
Isn't a proxy just a special case of a client? Not sure if this specific reference helps the reader or just creates confusion.

Reply:
i removed a few more mentions of "proxy", and left only the one at the very top "proxies"